### PR TITLE
refactor navmesh_utils SimpleVelocityControlEnv and de-duplicate

### PR DESCRIPTION
**Pull request by aclegg3**
## Motivation and Context

SimpleVelocityControlEnv and similar utils were duplicated with minor convention differences. This PR unifies the conventions and imports a single implementation in the oracle nav skills.

## How Has This Been Tested

CI

## Types of changes

- **\[Refactoring\]** Large changes to the code that improve its functionality or performance

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-lab/habitat/datasets/rearrange/navmesh_utils.py [hunk 1]
```diff
@@ -463,11 +465,12 @@ def act(self, trans: mn.Matrix4, vel: List[float]) -> mn.Matrix4:
 
         :return: The updated agent transformation matrix.
         """
+
         linear_velocity = vel[0]
         angular_velocity = vel[1]
         # Map velocity actions
         self.vel_control.linear_velocity = mn.Vector3(
-            [0.0, 0.0, -linear_velocity]
+            [linear_velocity, 0.0, 0.0]
```
Comment (jimmytyyang) [inline habitat-lab/habitat/datasets/rearrange/navmesh_utils.py:473]: hi alex, do you know why we do this at first place?

Comment (jimmytyyang) [reply habitat-lab/habitat/datasets/rearrange/navmesh_utils.py:473]: why [0.0, 0.0, -linear_velocity]?

**Review by jimmytyyang — APPROVED** [review]
LGTM

File: habitat-lab/habitat/datasets/rearrange/navmesh_utils.py (hunk 1 above)
Comment (aclegg3) [reply habitat-lab/habitat/datasets/rearrange/navmesh_utils.py:473]: Originally, this was alligned with Habitat's coordinate system. Typical Agent "forward" direction in local space is -Z.

**Pull request merged**